src/dodal/devices/areadetector/plugins/MXSC.py

```diff
@@ -1,16 +1,31 @@
+from typing import List, Tuple
+
+import numpy as np
 from ophyd import Component, Device, EpicsSignal, EpicsSignalRO, Kind, Signal
-from ophyd.status import Status, SubscriptionStatus
+from ophyd.status import StableSubscriptionStatus, Status
+
+from dodal.log import LOGGER
+
+
+def statistics_of_positions(positions: List[Tuple]):
+    x_coords, y_coords = np.array(positions).T
+
+    median = (int(np.median(x_coords)), int(np.median(y_coords)))
```

Comment:
Why do we want these to be integers?

Reply:
Good question. They're pixel values so we assumed they were integers later on for doing things like drawing grids and putting values into ispyb. Maybe I should do this conversion at the point we definitely need integers though

Reply:
Actually, there's so many places this could potentially break that I think conversion here makes sense, I will add a comment though.

Reply:
So I added type hinting, which makes it clearer that these things are valid `Pixel` locations and so must be ints. Doesn't really answer your question 100% but hopefully makes it clearer?